### PR TITLE
Add ssh option for direct host OS access

### PR DESCRIPTION
**Pull request by afitzek**
Use the `--host` (short `-s`) option in the ssh command to access
the host OS of the device.
Direct host OS is enabled for devices with Resin OS >= 2.7.5.

Change-Type: minor
Connects-To: #736
Signed-off-by: Andreas Fitzek <andreas@resin.io>
---- Autogenerated Waffleboard Connection: Connects to #736

### CODE REVIEW

File: doc/cli.markdown [hunk 1]
```diff
@@ -819,6 +820,10 @@ ssh gateway port
 
 increase verbosity
 
+#### --host, -H
+
+access host OS
```
Comment (thgreasi) [inline doc/cli.markdown]: I guess that we should document that this is
> enabled for devices with Resin OS >= 2.7.5.

File: lib/actions/ssh.coffee [hunk 2]
```diff
@@ -44,6 +45,11 @@ module.exports =
 			boolean: true
 			description: 'increase verbosity'
 			alias: 'v'
+	,
+			signature: 'host'
+			boolean: true
+			description: 'access host OS'
```
Comment (thgreasi) [inline lib/actions/ssh.coffee]: I guess that we should document that this is supported
> for devices with Resin OS >= 2.7.5.

File: lib/actions/ssh.coffee [hunk 3]
```diff
@@ -44,6 +45,11 @@ module.exports =
 			boolean: true
 			description: 'increase verbosity'
 			alias: 'v'
+	,
+			signature: 'host'
+			boolean: true
+			description: 'access host OS (for devices with Resin OS >= 2.7.5)'
+			alias: 'H'
```
Comment (pimterry) [inline lib/actions/ssh.coffee]: We try to share command line arguments in the CLI e.g. in https://github.com/resin-io/resin-cli/blob/master/lib/actions/command-options.coffee when they're used in multiple places - I think it'd be tidier to do that here for this and `local ssh`.

Notably, doing so highlights a difference: `local ssh` uses `-s` as the host alias, not `-H`. It'd be good to keep this consistent, and we can't change the `local ssh` format without a breaking change, so I think this should be `-s` too (though I wouldn't be surprised if we end up changing them both of together later, once we get to the big CLI redesign).

File: doc/cli.markdown [hunk 4]
```diff
@@ -808,6 +808,7 @@ Examples:
 	$ resin ssh 7cf02a6
 	$ resin ssh 7cf02a6 --port 8080
 	$ resin ssh 7cf02a6 -v
+	$ resin ssh 7cf02a6 -H
```
Comment (zwalchuk) [inline doc/cli.markdown]: Example still shows -H, should be -s I believe?

**Comment by thgreasi** [comment]
Woohoo!!
Would this also allow support agents connect to devices that have granted access?

**Comment by afitzek** [comment]
@thgreasi Yes.

**Comment by thgreasi** [comment]
Woohoo x2 !! :+1: :+1: :raised_hands: :heart: 
(We should add that to the support docs then.)

**Pull request merged**